src/rotary-dialer-element.ts

```diff
@@ -0,0 +1,280 @@
+              id="slots"
+              stroke="#fff"
+              fill-opacity="0.5"
+              fill="#D8D8D8"
+            ></path>
+          </g>
+          <circle id="container" fill-opacity=".5" fill="#070707" cx="132.5" cy="132.5" r="132.5" />
+          <g class="text" font-family="Monaco" font-size="21" fill="#FFF">
+            <text @mousedown=${() => this.down(0)} x="129" y="243">0</text>
+            <text @mousedown=${() => this.down(9)} x="78" y="230">9</text>
+            <text @mousedown=${() => this.down(8)} x="40" y="194">8</text>
+            <text @mousedown=${() => this.down(7)} x="28" y="145">7</text>
+            <text @mousedown=${() => this.down(6)} x="35" y="97">6</text>
+            <text @mousedown=${() => this.down(5)} x="72" y="58">5</text>
+            <text @mousedown=${() => this.down(4)} x="117" y="41">4</text>
+            <text @mousedown=${() => this.down(3)} x="168" y="47">3</text>
+            <text @mousedown=${() => this.down(2)} x="210" y="79">2</text>
+            <text @mousedown=${() => this.down(1)} x="230" y="126">1</text>
+          </g>
+          <path
+            d="M182.738529,211.096297 L177.320119,238.659185 L174.670528,252.137377 L188.487742,252.137377 L182.738529,211.096297 Z"
+            id="Triangle"
+            stroke="#979797"
+            fill="#D8D8D8"
+            transform="translate(181.562666, 230.360231) rotate(-22.000000) translate(-181.562666, -230.360231) "
```

Comment:
Can you please remove the transform?

Reply:
removing it will change the finger stop triangle position I need it a bit rotated :)